.github/snyk_pull_request_template.yaml

```
@@ -0,0 +1,4 @@
+commitMessage: |
+  Update dependencies for {{ snyk_project_name }}
+
+  Signed-off-by: Logan Nguyen <logan.nguyen@swirldslabs.com>
```

Ha, I missed this! I believe we can sign with Snyk’s signature instead of mine for better clarity. I usually sign because Snyk PRs don’t get signed.
```suggestion
  Signed-off-by: Swirlds Automation <swirlds-eng-automation@swirlds.com>
```
Check this commit for the signature example: https://github.com/hashgraph/hedera-json-rpc-relay/pull/3473/commits/f6d08111169be67b95c10c2514c83bf0358ee82a